webapp/test/test_repos.py

```diff
@@ -63,3 +67,15 @@ def test_schema(self):
         """Test schema of valid repos API response."""
         response = self.repo.process_list(api_version="v1", data=REPO_JSON)
         assert schemas.repos_schema.validate(response)
+
+    def test_modified_since(self):
+        """Test repos API with 'modified_since' property."""
+        repo = REPO_JSON_MODIFIED_SINCE.copy()
+        repo["modified_since"] = str(datetime.datetime.now().replace(tzinfo=pytz.UTC))
+        response = self.repo.process_list(api_version=1, data=repo)
+        assert tools.match(EMPTY_RESPONSE, response) is True
+
+        # without modified date
+        repo["repository_list"] = ["REPO-W/O-MODIFIED"]
```

Comment:
I don't know what does this trick means. If I want to use request "without modified data", I use json without `"modified_since"` field. Why do you use setting `"repository_list"` field to this strange value?

Reply:
@tkasparek @jdobes do you guys know what does the trick with `"REPO-W/O-MODIFIED"` mean?

Reply:
I removed it from repos, but in errata as I understand because of this
```python
# make errata_detail without ERRATA_UPDATED
errata_detail2 = self.cache.errata_detail[ERRATA_NAME]
errata_detail_list2 = list(errata_detail2)
errata_detail_list2[ERRATA_UPDATED] = None
self.cache.errata_detail["RHSA-W/O:MODIFIED"] = errata_detail_list2
```

we set especially to this **errata** `modified_date/updated` status to `None`, so after we can test specifically this errata with update status `None`